### DISCUSSION TITLE
Problem input stack not detected

### DISCUSSION THREAD

**Discussion by JackReacher007** [BrainEnergyLab/Inflammation-Index#29] [Fiji Plugin Issues]
Dear all,

I have used this amazing plugin with the provided reference image and it all works. However, it loosk like my input stack has the wrong format. After running Stack Preprocessing, this code is reported (see below). 

I could not find if the input stack needs to be formated in a certain way. Aside from my images being 1024^2 pixel and having 100um Z depth with 2um spacing, I could not idewntify why the plugin apparently does not recognice the stack,.

Would highly appreciate any input if someone has faced similar issues.

_Directories Input : E:\Microglia Morphology Analysis Plugin - ImageJ Example Directory and Input Data - Kopie\Working Directory\Input\
Directories Output : E:\Microglia Morphology Analysis Plugin - ImageJ Example Directory and Input Data - Kopie\Working Directory\Output\
Directories Done : E:\Microglia Morphology Analysis Plugin - ImageJ Example Directory and Input Data - Kopie\Working Directory\Done\
Directories Image Storage : E:\Microglia Morphology Analysis Plugin - ImageJ Example Directory and Input Data - Kopie\Image Storage Directory\
Directory E:\Microglia Morphology Analysis Plugin - ImageJ Example Directory and Input Data - Kopie\Working Directory\Input\ already exists
Directory E:\Microglia Morphology Analysis Plugin - ImageJ Example Directory and Input Data - Kopie\Working Directory\Output\ already exists
Directory E:\Microglia Morphology Analysis Plugin - ImageJ Example Directory and Input Data - Kopie\Working Directory\Done\ already exists
How many of the 'best' frames per Z plane do you want to include in the final Z plane image? : 1
How many frames do you want to include in the average projection of least blurry frames per Z plane? : 3
If you have already run the 'Stack QA' module, do you want to manually select frames to keep from images that failed automated selection QA in this run? : 0
What string should we search for in the Image Storage directory to find stacks to process? : Morphology
No image IDs to be manually processed
Retrieving .ini data
.ini file found at E:\Microglia Morphology Analysis Plugin - ImageJ Example Directory and Input Data - Kopie\Image Storage Directory\Scale_2PM_PCC.ini
x.pixel.sz =  0.4674
y.pixel.sz =  0.4674
z.spacing =  2
no.of.planes =  50
frames.per.plane =  8
No images to manually choose frames for
Image processing complete_

**Comment by CMol8** [comment]
Hi, I had the same issue. Probably you solved it too, but maybe my reply can be useful for future users.

In "_What string should we search for in the Image Storage directory to find stacks to process?_ : **Morphology**" this means that it will look for stacks containing the "Morphology" string in their name. If you leave this field empty, or if you specify a string which is actually in the names of your stacks, the problem should be solved.

I hope this helps.
